tasks/bootstrap-ssh.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Disallow SSH password authentication
```

Comment:
@goetzc Are you importing (calling) `bootstrap-ssh` somewhere in the bootstrap workflow? I didn't found

Reply:
Good catch, it was missing :+1: